tests/e2e/support/api/share/share.ts

```diff
@@ -179,3 +193,54 @@ export const addMembersToTheProjectSpace = async ({
 
   checkResponseStatus(response, 'Failed while adding members to project space')
 }
+
+export const createLinkShare = async ({
+  user,
+  path,
+  password,
+  name,
+  role,
+  spaceName
```

Comment:
can we add default value here?
```suggestion
  role = "view",
  spaceName = "personal"
```

Reply:
i don't think we can because according to our feature file, the role is sometimes edit also (`rename.feature:21`). So role can't have default value. 
In case of spaceName, we can have a default value for now since we're creating a public link of resources made in personal space. But what if need later for 'project' space? Shall I make it default?

Reply:
yes, default values are meant to be used when no optional params are provided (here: `role` and `spaceName`). if we want different role or spaceName other than default then we should be able to provide them as parameters in function call, right?

Reply:
right :)